tree/dataframe/src/RArrowDS.cxx

```diff
@@ -199,6 +199,14 @@ class ArrayPtrVisitor : public ::arrow::ArrayVisitor {
    using ::arrow::ArrayVisitor::Visit;
 };
 
+#if defined(__GNUC__) || defined(__clang__)
+#define likely(x)       __builtin_expect((x),1)
+#define unlikely(x)     __builtin_expect((x),0)
+#else
+#define likely(x)       x
+#define unlikely(x)     x
+#endif
+
```

Comment:
I think ROOT has its own definitions for these @pcanal ?

Reply:
Fixed.